package.json

```diff
@@ -30,6 +31,7 @@
     "gatsby-transformer-remark": "^6.3.2",
     "gatsby-transformer-sharp": "^5.3.1",
     "html-react-parser": "^1.2.7",
+    "js-search": "^2.0.0",
```

Comment:
The framework backing the search index. The searching happens all client-side, but I think we can statically compute the search index at build-time.